cli/src/crypto.js

```diff
@@ -1,18 +1,56 @@
 const crypto = require('crypto');
 
-const encrypt = (value, passphrase, algorithm = 'aes-256-ctr') => {
-    const cipher = crypto.createCipher(algorithm, passphrase);
-    const crypted = cipher.update(value, 'utf-8', 'hex');
+const ALGORITHM = 'aes-256-ctr';
+const KEY_BYTE_LENGTH = 32;
+const IV_LENGTH = 16;
 
-    return `${algorithm}:${crypted + cipher.final('hex')}`;
+const hexToBuffer = hex => new Buffer(hex, 'hex');
+const bufferToHex = buffer => buffer.toString('hex');
+
+const bufferize = (key, tryToCast = true) => {
+    if (!Buffer.isBuffer(key) || key.length !== KEY_BYTE_LENGTH) {
```

Comment:
Can you put the "check return" on top of the bufferize method and not use an inverse condition ?